src/madpack/utilities.py

```diff
@@ -189,6 +189,8 @@ def get_dbver(con_args, portid):
             # for Greenplum the 3rd digit is necessary to differentiate
             # 4.3.5+ from versions < 4.3.5
             match = re.search("Greenplum[a-zA-Z\s]*(\d+\.\d+\.\d+)", versionStr)
+        elif portid == 'cloudberry':
+            match = re.search("Cloudberry[a-zA-Z\s]*(\d+\.\d+\.\d+)", versionStr)
```

Comment:
`"Cloudberry[a-zA-Z\s]*(\d+\.\d+\.\d+)"` should be `"Apache Cloudberry[a-zA-Z\s]*(\d+\.\d+\.\d+)"` ?

I am not entirely sure about this.